sql/core/src/test/scala/org/apache/spark/sql/execution/metric/SQLMetricsSuite.scala

```diff
@@ -396,7 +396,9 @@ class SQLMetricsSuite extends SharedSparkSession with SQLMetricsTestUtils
       }
   }
 
-  test("SPARK-32629: ShuffledHashJoin(full outer) metrics") {
+  // TODO (SPARK-36272): Reenable this after we figure out why the expected size doesn't
+  // match after we adjust building's memory settings.
+  ignore("SPARK-32629: ShuffledHashJoin(full outer) metrics") {
```

Comment:
Thanks @viirya, I can also help look into this.

Reply:
🙏

Reply:
Filed https://github.com/apache/spark/pull/33494 now to log several data size metrics separately and keep debugging there.

Reply:
Thank you.